pyband/client.py

```diff
@@ -1,213 +1,114 @@
-        data = self._post("/txs", json={"tx": data, "mode": "block"})
-        if "code" in data:
-            code = int(data["code"])
-            error_log = data.get("raw_log")
-            log = []
-        else:
-            code = 0
-            error_log = None
-            log = data["logs"]
-
-        return TransactionBlockMode(
-            height=int(data["height"]),
-            tx_hash=HexBytes(bytes.fromhex(data["txhash"])),
-            gas_wanted=int(data["gas_wanted"]),
-            gas_used=int(data["gas_wanted"]),
-            code=code,
-            log=log,
-            error_log=error_log,
-        )
+    def get_oracle_script(self, id: int) -> oracle_type.OracleScript:
+        return self.stubOracle.OracleScript(oracle_query.QueryOracleScriptRequest(oracle_script_id=id)).oracle_script
 
-    def get_chain_id(self) -> str:
-        return self._get("/bandchain/chain_id")["chain_id"]
+    def get_request_by_id(self, id: int) -> oracle_query.QueryRequestResponse:
+        return self.stubOracle.Request(oracle_query.QueryRequestRequest(request_id=id))
 
-    def get_latest_block(self) -> Block:
-        return from_dict(
-            data_class=Block,
-            data=self._get("/blocks/latest"),
-            config=DACITE_CONFIG,
-        )
+    def get_reporters(self, validator: str) -> oracle_type.ReportersPerValidator.reporters:
+        return self.stubOracle.Reporters(oracle_query.QueryReportersRequest(validator_address=validator)).reporter
 
-    def get_account(self, address: Address) -> Optional[Account]:
-        try:
-            data = self._get("/cosmos/auth/v1beta1/accounts/{}".format(address.to_acc_bech32()))["account"]
-            return from_dict(
-                data_class=Account,
-                data=data,
-                config=DACITE_CONFIG,
-            )
-        except:
-            return None
-
-    def get_data_source(self, id: int) -> DataSource:
-        return from_dict(
-            data_class=DataSource,
-            data=self._get_result("/oracle/data_sources/{}".format(id)),
-            config=DACITE_CONFIG,
-        )
+    def get_latest_block(self) -> tendermint_query.GetLatestBlockResponse:
+        return self.stubCosmosTendermint.GetLatestBlock(tendermint_query.GetLatestBlockRequest())
 
-    def get_oracle_script(self, id: int) -> OracleScript:
-        return from_dict(
-            data_class=OracleScript,
-            data=self._get_result("/oracle/oracle_scripts/{}".format(id)),
-            config=DACITE_CONFIG,
-        )
+    def get_account(self, address: str) -> auth_type.BaseAccount:
+        account_any = self.stubAuth.Account(auth_query.QueryAccountRequest(address=address)).account
+        account = auth_type.BaseAccount()
+        if account_any.Is(account.DESCRIPTOR):
+            account_any.Unpack(account)
+            return account
+        else:
+            raise UnknownType("Unknown account")
```

Comment:
Based on old code, should it just return `None`?
```suggestion
            return None
```

Reply:
I agree that returns None make more sense.